.circleci/config.yml

```diff
@@ -59,9 +59,8 @@ jobs:
       - run:
           name: Run integration tests
           command: |
-            cp .env.sample .env
-            sudo chown -R $USER:$USER .
-            make integration_test_ci
+            sudo chown -R ubuntu tests/
```

Comment:
Any reason why you removed ```$USER:$USER```?

Reply:
Yeah it looked to me like that set it to `circleci` user, but when the container tries to write out to the integration test folder on the host machine its already owned by the `ubuntu` user?  You previously had the container execute some chowns to `seluser` which is `1001` which maps to `ubuntu` on the host, so this seemed to accomplish the same thing?

Reply:
What I had was probably unnecessary too